### PR TITLE
Axway Support Portal API - initial.

### PR DESCRIPTION
**Pull request by axdimitar**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Added documentation for Axway Support Portal API, part of AMPLIFY Foundation Services.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is clear and complete
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://techweb.axway.com/confluence/display/RDAPI/Best+practices+for+developer+documentation)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links
* [x] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [x] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [x] You have applied a production label if this needs a manual push to Zoomin
* [x] You have added it to the correct GitHub project
* [x] You have added it to the correct Sprint milestone
* [x] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue

### CODE REVIEW

File: content/en/docs/foundation/supportapi/apis/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: APIs & Reference
+linkTitle: APIs & Reference
+weight: 20
+date: 2020-03-05
+description: APIs & Reference.
+---
```
Comment (alexearnshaw) [inline content/en/docs/foundation/supportapi/apis/_index.md]: Actually, looking at the overall doc structure, I'd suggest removing this section altogether and having the "Formats" and "Methods" sections directly underneath "Axway Support Portal API documentation"
That would result in less clicks for a user to get to the information (best practice is for no more than 3 levels in the navigation)

File: content/en/docs/foundation/_index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: AMPLIFY Foundation documentation
+linkTitle: AMPLIFY Foundation
+weight: 170
+date: 2020-03-05
+description: Learn about AMPLIFY Shared services.
+---
+
```
Comment (alexearnshaw) [inline content/en/docs/foundation/_index.md]: Just an overall comment on the naming - I'm not quite sure I understand the difference between AMPLIFY Foundation and AMPLIFY Shared Services, and not sure users will either? At the moment you're using shared services in some places and foundation in others. I'd suggest sticking with one term only, whichever one is most meaningful to end users.

Comment (axdimitar) [reply content/en/docs/foundation/_index.md]: We decided to stick to AMPLIFY Shared Services in all places.
Apart from the other changes, I renamed the foundation directory to shared_services.

Not pushed anything yet though.

File: content/en/docs/foundation/supportapi/support/_index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
```
[the remark this answers is not in the document]
Comment (axdimitar) [reply content/en/docs/foundation/supportapi/support/_index.md]: Thanks, Alex! Makes sense!
Removed support and added a new section in methods/_index.md

**Comment by axdimitar** [comment]
Hi Alex,

I have just pushed my changes.
All of the comments/suggestions were addressed (hopefully - properly).
Please review.

Thanks!

Kind regards,
Dimitar

**Comment by alexearnshaw** [comment]
@axdimitar This is approved now and I will merge it shortly. It will then appear on https://axway-open-docs.netlify.com/

I hope to push it to production https://docs.axway.com/bundle/axway-open-docs/ later today but there's a couple of things I need to change there first, to classify it with the tag "AMPLIFY Shared Services" public visibility. Will let you know when it's done.

**Comment by axdimitar** [comment]
Thank you for your time and help, @alexearnshaw !

**Comment by alexearnshaw** [comment]
@axdimitar The doc is published now at https://docs.axway.com/bundle/axway-open-docs/page/docs/shared_services/index.html

**Comment by axdimitar** [comment]
Thank you, @alexearnshaw !

**Pull request merged**